src/pybind/mgr/test_orchestrator/module.py

```diff
@@ -259,7 +259,7 @@ def run(all_hosts):
             # type: (List[orchestrator.HostSpec]) -> None
             drive_group.validate()
             if drive_group.placement.host_pattern:
-                if not drive_group.placement.pattern_matches_hosts([h.hostname for h in all_hosts]):
+                if not drive_group.placement.filter_matching_hosts(self.get_hosts):
```

Comment:
`self.get_hosts` returns a completion, not a `List[HostSpec]`

Reply:
done